ee/clickhouse/migrations/0009_person_deleted_column.py

```diff
@@ -0,0 +1,9 @@
+from infi.clickhouse_orm import migrations
+
+from ee.clickhouse.sql.person import KAFKA_PERSONS_TABLE_SQL
+
+operations = [
+    migrations.RunSQL("DROP TABLE kafka_person"),
+    migrations.RunSQL("ALTER TABLE person ADD COLUMN IF NOT EXISTS is_deleted Boolean DEFAULT 0"),
+    migrations.RunSQL(KAFKA_PERSONS_TABLE_SQL),
```

Comment:
Same with this. Once you delete the table on the cluster you'll want to recreate it on the cluster.

DDLs will require `ON CLUSTER posthog` in general. Without it the cluster would end up in an inconsistent state.